src/dom-to-image-more.js

```diff
@@ -269,10 +335,10 @@
         return Promise.resolve(node)
             .then(makeNodeCopy)
             .then(function (clone) {
-                return cloneChildren(getParentOfChildren(node), clone);
+                return cloneChildren(clone, getParentOfChildren(node));
```

Comment:
Swap the order of the arguments to follow normal `.then()` construction _(i.e. optional/additional arguments on the right)_